qualang_tools/optimal_weights/StateDiscriminator.py

```diff
@@ -14,12 +14,12 @@ class StateDiscriminator:
     The state discriminator is a class that generates optimized measure procedure for state discrimination
     of a multi-level qubit.
     .. note:
-        Currently only 3-states discrimination is supported. The setup assumed here includes IQ mixer both in the up-
+        Currently only 2-state discrimination is supported. The setup assumed here includes IQ mixer both in the up-
         and down-conversion of the readout pulse.
     """
 
     def __init__(
-        self, qmm, config, update_tof, rr_qe, path, meas_len, smearing, lsb=False
+        self, qmm, config, update_tof, rr_qe, rr_pulse, path, meas_len, smearing, lsb=False
```

Comment:
I would change `rr_qe` to either: `resonator_el` or `rr_el`.
Also, you need to add `rr_pulse` to the docstring

Reply:
thanks